### ISSUE TITLE
Lambda function as a Docker Image?

### ISSUE THREAD

**Issue by juanmarin96** [newrelic/serverless-newrelic-lambda-layers#270]
Does the plugin work with lambda functions deployed as a Docker Image?

**Comment by mrickard** [comment]
We haven't tested the plugin with Lambda functions deployed as Docker images, and I'd suspect that it'd throw errors. 

On the other hand, customers have installed our agent/extension layers (https://layers.newrelic-external.com/) into Lambdas deployed as Docker images, following the procedure outlined by AWS here: https://docs.aws.amazon.com/lambda/latest/dg/images-create.html. You'd want to be careful to set the handler value and environment variables correctly.

**Comment by cabal-daniel** [comment]
Yeah this project doesn't work with images. Which is unfortunate as only images are able to support 10G size on AWS. I hope it gets planned soon.

**Comment by chistyakov** [comment]
I'd like to propose the addition of Docker images pre-integrated with NewRelic's Lambda layer. At present, our team finds itself consistently building and hosting our own Docker images whenever there's a new version of the Lambda layer. This process can be quite cumbersome.

I've observed that NewRelic's competitor, Sentry, offers this convenience for their users. Here's a reference to how they've implemented it: [Sentry AWS Lambda Container Image Documentation](https://docs.sentry.io/platforms/python/guides/aws-lambda/container-image/).

Providing such a feature would greatly streamline our deployment process and enhance user experience. I believe other users would benefit from this addition as well.

Thank you for considering this enhancement!

**Comment by chaudharysaket** [comment]
We have released support for containerized lambda. Please find the New Relic [docs](https://docs.newrelic.com/docs/serverless-function-monitoring/aws-lambda-monitoring/instrument-lambda-function/containerized-images/). You can find the ECR images [here](https://gallery.ecr.aws/newrelic-lambda-layers-for-docker?page=1). 
Thank you!

**Comment by chistyakov** [comment]
thank you!

**Comment by chistyakov** [comment]
Is it possible to add the newrelic-agent version to the docker-image tag?

I want to freeze the image version to prevent unexpected updates.
